modules/desktop/src/routes/packages/[slug]/+page.svelte

```diff
@@ -30,7 +30,7 @@
   // let reviews: Review[];
   $: bottles = pkg?.bottles || [];
   $: versions = [...new Set(bottles.map((b) => b.version))];
-  $: readme = pkg?.readme || { data: "", type: "md" };
+  $: readme = pkg?.readme || { data: "", type: "html" };
```

Comment:
@getneil post processing is still necessary to handle <a> correctly to open on the default browser